packages/expo/android/src/rn78/main/expo/modules/ExpoReactHostFactory.kt

```diff
@@ -58,11 +56,7 @@ object ExpoReactHostFactory {
       get() = reactNativeHostWrapper.jsMainModuleName
 
     override val jsRuntimeFactory: JSRuntimeFactory
-      get() = if (reactNativeHostWrapper.jsEngineResolutionAlgorithm == JSEngineResolutionAlgorithm.HERMES) {
-        HermesInstance()
-      } else {
-        JSCInstance()
-      }
+      get() = HermesInstance()
```

Comment:
We might need to test this with https://github.com/react-native-community/javascriptcore